src/NodeFactory/ClassWithPublicPropertiesFactory.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Nette\NodeFactory;
+
+use PhpParser\Node;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+use PhpParser\Node\NullableType;
+use Symplify\Astral\ValueObject\NodeBuilder\ClassBuilder;
+use Symplify\Astral\ValueObject\NodeBuilder\NamespaceBuilder;
+use Symplify\Astral\ValueObject\NodeBuilder\PropertyBuilder;
+use Symplify\Astral\ValueObject\NodeBuilder\TraitUseBuilder;
+
+
+/**
+ * @TODO tests
+ */
+final class ClassWithPublicPropertiesFactory
```

Comment:
this could be used later also for Template classes (https://latte.nette.org/en/type-system). Maybe it is generic enough to be part of rector-src core, not just rector-nette

Reply:
>  Maybe it is generic enough to be part of rector-src core

:+1:

Reply:
Ok I will prepare PR for this in core but what about the rest? Imho it is nette specific and should stay here

Reply:
What is nette-specific should remain here, yes :+1: